src/bicep/mlz.bicep

```
@@ -252,6 +252,58 @@ module sharedServicesVirtualNetworkPeering './modules/spokeNetworkPeering.bicep'
   }
 }
 
+module hubPolicyAssignment './modules/policyAssignment.bicep' = {
+  name: 'policyAssignement'
+  scope: resourceGroup(hubSubscriptionId, hubResourceGroupName)
+  dependsOn: [
+    hubResourceGroup
+    logAnalyticsWorkspace
+  ]
+  params: {
+    builtInAssignment: policy
+    logAnalyticsWorkspaceName: logAnalyticsWorkspace.outputs.name
+    logAnalyticsWorkspaceResourceGroupName: operationsResourceGroupName
+  }
+}
+
+module operationsPolicyAssignment './modules/policyAssignment.bicep' = {
+  name: '${operationsResourceGroupName}-policyAssignment'
+  scope: resourceGroup(operationsSubscriptionId, operationsResourceGroupName)
+  params: {
+    builtInAssignment: policy
+    logAnalyticsWorkspaceName: logAnalyticsWorkspace.outputs.name
+    logAnalyticsWorkspaceResourceGroupName: operationsResourceGroup.name
```

Was getting a deployment failure on a non-existent RG. Since `operationsResourceGroup` is a module, the name of this object (`operationsResourceGroup.name`) is of the deployment. We'd need to refer to the module output to get the resource group's resource name:

```suggestion
    logAnalyticsWorkspaceResourceGroupName: operationsResourceGroup.outputs.name
```